src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -549,39 +627,113 @@ public void SizeOf_04()
                 Diagnostic(ErrorCode.ERR_NotConstantExpression, "sizeof(nuint)").WithArguments("Program.D").WithLocation(6, 19));
         }
 
-        // PROTOTYPE: PEVerify: "[ : MyInt::ToString][mdToken=0x6000005][offset 0x00000001] Cannot change initonly field outside its .ctor."
-        // CodeGenerator.EmitCallExpression() is not expecting a System.ValueType without an overload of ToString().
-        [Fact(Skip = "PEVerify failure")]
+        [Fact]
         public void ReadOnlyField_VirtualMethods()
         {
             string source =
-@"class MyInt
+@"using System;
+using System.Linq.Expressions;
+class MyInt
 {
     private readonly nint _i;
-    internal MyInt(nint i) => _i = i;
-    public override string ToString() => _i.ToString();
+    internal MyInt(nint i)
+    {
+        _i = i;
+    }
+    public override string ToString()
+    {
+        return _i.ToString();
+    }
+    public override int GetHashCode()
+    {
+        return ((Func<int>)_i.GetHashCode)();
+    }
+    public override bool Equals(object other)
+    {
+        return _i.Equals((other as MyInt)?._i);
+    }
+    internal string ToStringFromExpr()
+    {
+        Expression<Func<string>> e = () => ((Func<string>)_i.ToString)();
+        return e.Compile()();
+    }
+    internal int GetHashCodeFromExpr()
+    {
+        Expression<Func<int>> e = () => _i.GetHashCode();
+        return e.Compile()();
+    }
 }
 class Program
 {
     static void Main()
     {
         var m = new MyInt(42);
-        System.Console.WriteLine(m);
+        Console.WriteLine(m);
+        Console.WriteLine(m.GetHashCode());
+        Console.WriteLine(m.Equals(null));
+        Console.WriteLine(m.ToStringFromExpr());
+        Console.WriteLine(m.GetHashCodeFromExpr());
     }
 }";
             var comp = CreateCompilation(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.RegularPreview);
-            var verifier = CompileAndVerify(comp, expectedOutput: "42");
+            // PEVerify fails with "MyInt::ToString][mdToken=0x6000007][offset 0x00000001] Cannot change initonly field outside its .ctor."
+            var verifier = CompileAndVerify(comp, verify: Verification.Skipped, expectedOutput:
+$@"42
+{42.GetHashCode()}
+False
+42
+{42.GetHashCode()}");
+            // PROTOTYPE: Should we generate the following instead, which would correspond to the equivalent
+            // code generated for `_i.ToString()` if `_i` was `int`? If NativeIntegerTypeSymbol.GetMembers() included
+            // members for ToString(), Equals(), GetHashCode(), that would fix this issue and the PEVerify failure above.
```

Comment:
>members for ToString(), Equals(), GetHashCode(), that would fix this issue and the PEVerify failure above. [](start = 15, length = 106)

Consider explicitly stating that the code we generate right now is equivalent to a code for a regular structure that doesn't override ToString()